src/main/java/zpi/config/BeansConfig.java

```diff
@@ -0,0 +1,15 @@
+package zpi.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.crypto.password.PasswordEncoder;
+
+@Configuration
+public class BeansConfig {
+
+    @Bean
+    public PasswordEncoder passwordEncoder() {
+        return new BCryptPasswordEncoder();
```

Comment:
Wydaje mi się, że nie potrzebujemy tego dawać w osobnej klasie i w SecurityConfigu to by mogło być, ale to bardziej takie moje wrażenie. Zrób jak uważasz

Reply:
Ta zbedna klasa, zostalo ze mszy ;)